modules/core/shared/src/main/scala/com/github/fsanaulla/chronicler/core/model/InfluxPoint.scala

```diff
@@ -92,7 +92,13 @@ final case class Point(
   def serialize: String = {
     val sb = new StringBuilder()
 
-    sb.append(measurement.escapeMeas).append(",")
+    sb.append(measurement.escapeMeas)
+
+    tags match {
```

Comment:
I think we can rewrite this expression: 
```
if (tags.nonEmpty) {
      val tgs = tags
        .map(tag => tag.key.escapeKey + "=" + tag.value.escapeKey)
        .mkString(",")

      sb.append("," + tgs)
    }
```
Using simple `if` expresison, without rudimental operation on empty list

Reply:
nice idea. do you want to change it? or should i update my pull-request? 
in case i should update, i will also add a unit test.

Reply:
If you can do it, it will be great! Thank you.